css/components/works.css

```diff
@@ -0,0 +1,132 @@
+.works {
+    display: inline-block;
+    text-align: center;
+    position: relative;
+    padding-top: 40px;
+}
+h2 {
```

Comment:
it would be better if you would use `.work h2` selector for header, because now updates header went to the center